Electron/Electron12/protected/jsdefender.config.json

```diff
@@ -30,7 +30,7 @@
         },
         "propertyIndirection": true,
         "selfDefending": true,
-        "stringLiterals": true,
+        "stringLiterals": false,
```

Comment:
why is this turned off? do we have issue with string literals enabled?

Reply:
When stringLiterals is enabled then css definitions are broken.
It happens in both versions of jsdefender: official/current version 2.6.0 and in new one.

Reply:
basically, you can simply remove desired files from obfuscation